nixos/tests/gitlab.nix

```diff
@@ -32,6 +32,94 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : with lib; {
         databasePasswordFile = pkgs.writeText "dbPassword" "xo0daiF4";
         initialRootPasswordFile = pkgs.writeText "rootPassword" initialRootPassword;
         smtp.enable = true;
+        registry = {
+          enable = true;
+          certFile = pkgs.writeText "registry.crt" ''
```

Comment:
can you create certs on the fly, like in `taskserver.nix`? This should be way more readable  than dumping ~80lines here ;-)